packages/app-permissions/src/index.js

```diff
@@ -33,7 +33,12 @@ export const mergeGrantsToDetails = (
     ...app, // Manifest requirements overrides
     // Special case
     HTTPS_REQUEST: {
-      granted: app.HTTPS_REQUEST.granted.concat(user.HTTPS_REQUEST.granted),
+      granted: [
+        ...new Set([
```

Comment:
This shouldn't be done here, the `PermissionsGrants` themselves should be sanitised

Reply:
I don't think I understand... whether or not the PermissionsGrants are sanitized, they still need a set union to remove duplicates when merging.

Reply:
There should never be a domain present both in `app` and `user` `PermissionGrants`, if so it means there is an issue with the way they are managed.

Reply:
I'll revert.